harvard-lts/CURIOSity
# Delete rake task

**A rake task for deletes**
* * *


# What does this Pull Request do?
A first-pass of a rake task for deleting items from Curiosity

# How should this be tested?

I recorded a demo of how to test on trial, summary is:

- spin up container or use dev
- make sure you have a collection with stuff in it
- go to the Items list and uncheck a few items you want deleted
- put the IDs for those in a file (the IDs are the end part of the URL); the easiest way to do this is IN the container and do the following:
touch deletes.txt
printf "ID1\nID2\nID3\n..."

- run the deletes: bundle exec rake curiosity:delete_items[deletes.txt]
- when this finishes, run a re-index
- items should be gone!

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n but if we can think of how to test it I'd be game
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
